### ISSUE TITLE
Is MNIST linear in its bones

### ISSUE THREAD

**Issue by dorimedini** [dorimedini/kardashigans#60]
We think FCN (N-layer FC nets) on MNIST will just try to become 1-layer because MNIST is a linear-solvable problem. Try, say, fitting a 5 layer FC net with all but the first layer frozen.

**Comment by galshachaf** [comment]
But we know it's linear-solvable. Amir mentioned it on the first lecture.

**Comment by dorimedini** [comment]
Yes, but that's not exactly the question... from what I understand we also want to know if, given a more powerful model topology, the resulting trained model is in fact just a one-layer linear classifier with added noise before output. Right? @noam
